raiden-dapp/src/model/types.ts

```diff
@@ -8,6 +8,10 @@ export interface Token {
   readonly name?: string;
 }
 
+export interface Presence {
+  [address: string]: boolean;
```

Comment:
Please, don't store only the boolean, but the entire presences object, as it contains other useful information, like last time the user was seen online.

Reply:
I feel like if the information isn't accessed and not part of the reqs it doesn't make sense to include it, as it can easily be extended later on, if ever needed

Reply:
Ok, I'd have written the requirements differently, to also include this as part of the new feature, but you're right, it's not there and thus it's acceptable to only include the bare minimum data you need, although it'd be more complete the other way.